src/cljs/org/broadinstitute/firecloud_ui/page/workspaces_list.cljs

```diff
@@ -208,30 +208,41 @@
                        assoc :error-message status-text)))}))})
 
 
-(defn- get-workspace-id-from-nav-segment [segment]
-  (when-not (clojure.string/blank? segment)
-    (let [[ns n] (clojure.string/split segment #":")]
-      {:namespace ns :name n})))
+(defn- create-breadcrumbs-from-hash [hash]
+  (let [segments (split hash #"/")]
+    (map-indexed
+      (fn [index segment]
+        (case index
+          0 {:text "Workspaces" :href "#workspaces"}
+          1 {:text (replace (js/decodeURIComponent segment) ":" "/")}
```

Comment:
Without an :href here, there's no way to use the breadcrumbs to get back to the workspace summary? Is that the wonky second link issue you mentioned?